e2e/testcases/policy_dir_test.go

```diff
@@ -50,30 +50,24 @@ func TestPolicyDirUnset(t *testing.T) {
 	nt.Must(rootSyncGitRepo.Copy("../../examples/acme/namespaces", "."))
 	nt.Must(rootSyncGitRepo.Copy("../../examples/acme/system", "."))
 	nt.Must(rootSyncGitRepo.CommitAndPush("Initialize the root directory"))
-	if err := nt.WatchForAllSyncs(); err != nil {
-		nt.T.Fatal(err)
-	}
+	nt.Must(nt.WatchForAllSyncs())
 
-	nomostest.SetPolicyDir(nt, configsync.RootSyncName, "")
-	err := nt.WatchForAllSyncs(nomostest.WithSyncDirectoryMap(map[types.NamespacedName]string{nomostest.DefaultRootRepoNamespacedName: "."}))
-	if err != nil {
-		nt.T.Fatal(err)
-	}
+	nomostest.SetRootSyncGitDir(nt, rootSyncID.Name, "")
+	nomostest.SetExpectedSyncPath(nt, rootSyncID, ".") // TODO: OCI default sync path?
```

Comment:
TIL `controllers.DefaultSyncDir` and `gitproviders.DefaultSyncDir` are different, confusingly so.

Reply:
`controllers.DefaultSyncDir` is used by the reconciler in prod, while `gitproviders.DefaultSyncDir` is only used for testing.

Reply:
Replaced most of these with two in the OCI RSync constructors, tho those only work (for now) if the source was Git or OCI before. I'll add OCISourceSync construction in a followup PR.

This on specifically is an exception, since it's changed after construction, but I changed it to use the constant.